doc/api/n-api.md

```diff
@@ -6126,14 +6126,18 @@ NAPI_EXTERN napi_status napi_adjust_external_memory(node_api_basic_env env,
 * `[in] env`: The environment that the API is invoked under.
 * `[in] change_in_bytes`: The change in externally allocated memory that is kept
   alive by JavaScript objects.
-* `[out] result`: The adjusted value
+* `[out] result`: The adjusted value. This value should reflect the total amount
+  which was been adjusted based on calls to `napi_adjust_external_memory` so far.
+  For example, if there has been a request for +1000 and a request for -500 it should
+  return 500.
 
 Returns `napi_ok` if the API succeeded.
 
-This function gives V8 an indication of the amount of externally allocated
-memory that is kept alive by JavaScript objects (i.e. a JavaScript object
-that points to its own memory allocated by a native addon). Registering
-externally allocated memory will trigger global garbage collections more
+This function gives the runtime an indication of the amount of externally
+allocated memory that is kept alive by JavaScript objects
+(i.e. a JavaScript object that points to its own memory allocated by a
+native addon). Registering externally allocated memory may, but is not
+guaranteed to, trigger global garbage collections more
 often than it would otherwise.
```

Comment:
```suggestion
often than it would otherwise.

This function is expected to be invoked symmetrically. If it is invoked with +512KB,
it is expected to invoke -512KB in sum at a later time.
```